src/Model/fn.ts

```diff
@@ -90,6 +90,10 @@ declare module './Model' {
      * It's not recommended to use this in most cases. Instead, to share reactive functions,
      * have the components import a shared function to pass to `model.start`.
      *
+     * @deprecated The use of named funcitons is deprecated. With typescript and modern tooling
+     * you get better type information, code navigation, and refactoring support that is lost
+     * when using named functions.
```

Comment:
Typo `funcitons` -> `functions`. And it'd also be good to recommend a replacement, something like:

```suggestion
     * @deprecated The use of named functions is deprecated. Instead, to share a reactive function,
     * you should export it and then require/import it into each file that needs to use it.
```